server/sonar-server/src/main/java/org/sonar/server/user/ws/CurrentAction.java

```diff
@@ -95,6 +98,7 @@ private CurrentWsResponse toWsResponse(DbSession dbSession, String userLogin) {
       .setPermissions(Permissions.newBuilder().addAllGlobal(getGlobalPermissions()).build())
       .setShowOnboardingTutorial(!user.isOnboarded());
     setNullable(emptyToNull(user.getEmail()), builder::setEmail);
+    setNullable(emptyToNull(user.getEmail()), u -> builder.setAvatar(avatarResolver.create(user)));
```

Comment:
Do not forget to update example https://github.com/SonarSource/sonarqube/blob/master/server/sonar-server/src/main/resources/org/sonar/server/user/ws/current-example.json#L5